README.md

```diff
@@ -1 +1,72 @@
-# Bratislava-Rust-Meetup
\ No newline at end of file
+Hello, for English, scroll below.
+Ahojte, pre Slovenčinu čítajte ďalej.
+
+---
+
+# Bratislava-Rust-Meetup (slovensky)
+
+Repozitár s informáciami o Bratislava-Rust-Meetup skupine.
+
+Ďalšie stretnutie: Bude ohlásené čoskoro.
+
+## O skupine
+
+Zaujímame sa o nový programovací jazyk Rust od Mozilly. Vítame všetkých
+nadšencov od nováčikov až po expertov. Aj keď Vás Rust zatiaľ len zaujal. Chceme sa stretávať, učiť sa o Ruste a pracovať na Rust projektoch všetkých druhov.
+
+## Debaty
+
+Radi si vypočujeme Váš názor!
+
+- Čo by ste chceli odprezentovať ostatným?
+- O čom by ste sa chceli dozvedieť viac?
+- S čím potrebujete pomoc?
+
+## Ďalšie stretnutia
+
+Budú ohlásené.
+
+## Stretnutia
+
+### Initial meet and greet Rust Meetup
+
+Kde: Nivy Tower 6th floor
+
+## Odkazy
+
+[Meetup](https://www.meetup.com/bratislava-rust-meetup-group/)
+
+# Bratislava-Rust-Meetup (english)
+
+Repository with information about the Bratislava Rust Meetup group.
```

Comment:
Meetings are conducted in Slovak, but we'd be happy to get in touch with you even if you can't understand Slovak.